uaclient/cli.py

```diff
@@ -908,9 +906,40 @@ def action_config_unset(args, *, cfg, **kwargs):
     return 0
 
 
+def _create_enable_disable_unattached_msg(args):
```

Comment:
I think the function should receive the command and services as an argument instead, just so we can better decouple it from the cli args